webcompat/templates/home-page/form.html

```diff
@@ -159,5 +160,18 @@ <h3 class="wc-Title--s">Report Site Issue</h3>
       </div>
       {{ form.hidden_tag() }}
     </form>
+  <!-- The Modal -->
+  <div id="imageEditor" class="modal is-hidden">
+
+    <!-- Modal content -->
+    <div class="modal-content">
+      <span class="close js-edit-img-close-btn">&times;</span>
+      <div class="wc-Image-Editor-toolbar">
+        <div class="js-highlight-button">Highlight Rect</div>
```

Comment:
@miketaylr agreed. I did some sketchy wireframes in the am.

https://docs.google.com/document/d/11KZICiXwfU8znYtkDzK3y7MFYyEznWKKQXzknLwrb1k/edit?usp=sharing